packages/chat-component-bindings/src/messageThreadSelector.ts

```diff
@@ -203,7 +203,7 @@ export const messageThreadSelector: MessageThreadSelector = createSelector(
     updateMessagesWithAttached(convertedMessages);
     return {
       userId,
-      showMessageStatus: !isLargeGroup,
+      showMessageStatus: true,
```

Comment:
What if we set this as undefined? Will that be default on or off?

Reply:
'Set it as undefined' means not to set it

Reply:
> 'Set it as undefined' means not to set it

Setting it to undefined will cause it to use the default value provided in MessageThread.ts which is false which is contradictory with our current behaviour.